### ISSUE TITLE
Does this script work with php 7?

### ISSUE THREAD

**Issue by karlkras** [c9/c9.ide.run.debug.xdebug#4]
So using laravel 5.5 I need php 7 which I've installed... do is this script as you show here:
Cloud9 core plugin for Xdebug and other DBGP debuggers.

to install xdebug for php use

sudo apt-get update
sudo apt-get install -y php5-dev
sudo pecl install xdebug
sudo mkdir -p /etc/php5/mods-available
echo "; Xdebug extension installed by Cloud9
zend_extension=xdebug.so
xdebug.remote_enable=1
" | sudo tee --append /etc/php5/mods-available/xdebug.ini
sudo php5enmod xdebug

work under php 7 ? if so, it doesn't appear to be.

**Comment by collierscott** [comment]
I am getting:

Warning: can’t enable debugging because xdebug plugin for php is not installed.
see https://github.com/c9/c9.ide.run.debug.xdebug for details.

php -v

PHP 7.1.13-1+ubuntu14.04.1+deb.sury.org+1 (cli) (built: Jan 16 2018 15:37:12) ( NTS )
Copyright © 1997-2017 The PHP Group
Zend Engine v3.1.0, Copyright © 1998-2017 Zend Technologies
with Zend OPcache v7.1.13-1+ubuntu14.04.1+deb.sury.org+1, Copyright © 1999-2017, by Zend Technologies
with Xdebug v2.5.5, Copyright © 2002-2017, by Derick Rethans

xdebug.ini contents

; Xdebug extension installed by Cloud9
zend_extension=xdebug.so
xdebug.remote_enabled=1

I am trying to develop a Symfony 4 project and need PHP 7.1.

**Comment by kattunga** [comment]
In PHP 7.0 Adjust names to php 7.0. I didn't tested PHP 7.1
```
sudo apt-get update
sudo apt-get install -y php7.0-dev
sudo pecl install xdebug
echo "; Xdebug extension installed by Cloud9
zend_extension=xdebug.so
xdebug.remote_enable=1
" | sudo tee --append /etc/php/7.0/mods-available/xdebug.ini
sudo phpenmod xdebug
```

**Comment by collierscott** [comment]
Sorry, I should have stated. I did substitute 7.1.

Steps I took:

sudo add-apt-repository ppa:ondrej/php -y
sudo apt-get update -y

sudo apt-get install php7.1-curl php7.1-dev php7.1-gd php7.1-intl php7.1-mcrypt php7.1-json php7.1-mysql php7.1-opcache php7.1-bcmath php7.1-mbstring php7.1-soap php7.1-xml php7.1-xdebug

sudo a2dismod php7.0
sudo apt-get install libapache2-mod-php7.1 -y

echo "; Xdebug extension installed by Cloud9
zend_extension=xdebug.so
xdebug.remote_enable=1
" | sudo tee --append /etc/php/7.1/mods-available/xdebug.ini
sudo phpenmod xdebug

![capture](https://user-images.githubusercontent.com/24322760/35779631-c7e089f0-09d0-11e8-9593-eb3b4557e6cb.PNG)

**Comment by karlkras** [comment]
Yeah, they're working on it. [see here](https://community.c9.io/t/xdebug-plugin-not-installing-in-workspace/22142)
